### PR TITLE
feat: Mongodb collection

### PR DESCRIPTION
**Pull request by antonioT90**
### List of changes

added mongoDb collection creation together with lock feature

### Type of changes

- [x] Add new module
- [ ] Update existing module
- [ ] Remove existing module

### Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: cosmosdb_mongodb_collection/variables.tf [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# General Variables
+variable "name" {
+  type = string
```
Comment (diegolagospagopa) [inline cosmosdb_mongodb_collection/variables.tf:3]: Please add a description

File: cosmosdb_mongodb_collection/variables.tf [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+// CosmosDB Mongodb Collection specific variables
+variable "cosmosdb_mongo_account_name" {
+  type = string
+}
+
+variable "cosmosdb_mongo_database_name" {
+  type = string
+}
+
+variable "ttl" {
```
Comment (diegolagospagopa) [inline cosmosdb_mongodb_collection/variables.tf:26]: please add a description & a better name for variable to be more simple to understand how to use: `default_ttl_seconds `

**Review by diegolagospagopa — CHANGES_REQUESTED** [review]
* Please add the description for the variable where no present, to be more simple to use the module.
* Add a Readme (you can copy from kubernetes) to explain what this module does, and an example how to use it

**Comment by antonioT90** [comment]
@diegolagospagopa the static analysis failed due to tfdocs generating a different file from the one wrote by me.
I was not able to create it automatically, so I tried to create and fill it in accordance, but I have written something different from the automatic tool result and I'm not able to see it, could you please generate automatically them :) ?

**Comment by diegolagospagopa** [comment]
> @diegolagospagopa the static analysis failed due to tfdocs generating a different file from the one wrote by me.
I was not able to create it automatically, so I tried to create and fill it in accordance, but I have written something different from the automatic tool result and I'm not able to see it, could you please generate automatically them :) ? 

Fixed, the problem was related to an end tag not present

**Review by diegolagospagopa — APPROVED** [review]

**Pull request merged**